src/components/Admin/Index.tsx

```diff
@@ -19,7 +19,6 @@ import { useTranslation } from "react-i18next"
 import { ReactComponent as Researcher } from "../../icons/Researcher.svg"
 import { ReactComponent as DataPortalIcon } from "../../icons/DataPortal.svg"
 import { MuiThemeProvider, makeStyles, Theme, createStyles, createMuiTheme } from "@material-ui/core/styles"
-import locale_lang from "../../locale_map.json"
```

Comment:
We should find a way to source the language names from the library itself so we can delete the JSON file.